src/styles/global/print.scss

```diff
@@ -1,8 +1,96 @@
 //
 // CUSTOMIZED PRINT STYLES
+@import 'variables';
 
 @media print {
+  @page {
+    margin-top: 100px;
+  }
+
+  @supports (-moz-appearance: none) {
+    .logo {
+      margin-left: 25px !important;
+    }
+  }
+
   a[href]::after {
     content: none !important;
   }
+
+  button {
```

Comment:
Please move the button styling down to where the other button styling is applied. You should first have all items listed which are not displayed and afterwards the elements where a certain visual styling is applied. It makes it easier to read the CSS file.